### DISCUSSION TITLE
"Save password" option & GUI adjustments

### DISCUSSION THREAD

**Discussion by CLU91** [rustdesk/rustdesk-server-pro#149] [Ideas]
Hello, 

we are using Rustdesk in our company environment at the moment. We have some suggestions for things that we noticed: 

First we would love to have an option, to disable "save password" with the permanent password. The permanent password option is a security feature for some servers. Some employees choose the easy way and save password in the their connection. Thats something i would like to control as admin. Maybe you can have a setting under security settings > permanent password > "Disable store permanent password" or similar description....

Second thing, we currently have about 50 devices in our adress book. It start to look confusing with all these different colors and most of all the indication of an TAG in the right upper corner looks very similar to the online/offline status right next to the name. Especially when you choose colors like green and orange. Here a screenshot for you to see what i mean:

<img src="https://i.imgur.com/yVQhNWF.jpeg"></img>

Our adressbook looks like a christastree meanwhile. You might consider changing the TAG style or the way the online/offline status is shown. 

regards

**Comment by AussieMakerGeek** [comment]
+1 on 'Disable save password' option. There also does not seem to be an option to delete the saved password if you save it.
